src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -170,6 +170,8 @@ private static String injectRealAccounts(String jsonString) {
                 .replace("${test.student1}", TestProperties.inst().TEST_STUDENT1_ACCOUNT)
                 .replace("${test.student2}", TestProperties.inst().TEST_STUDENT2_ACCOUNT)
                 .replace("${test.instructor}", TestProperties.inst().TEST_INSTRUCTOR_ACCOUNT)
+                .replace("${test.unreg}", TestProperties.inst().TEST_UNREG_ACCOUNT)
+                .replace("${test.admin}", TestProperties.inst().TEST_ADMIN_ACCOUNT)
```

Comment:
This method is used during insertion of the json data files into the database. The truncated admin should only be used during display, so it should not make sense to insert a truncated version of the admin's email into the database. 

This code is used for the data json files that inserts data based on the test accounts. e.g. InstructorCourseJoinConfirmationPageUiTest.json
![substituedinstructors](https://cloud.githubusercontent.com/assets/2216119/9897790/1764e9f0-5c7f-11e5-8fef-81763e1ca0b5.PNG)